rooms/models.py

```diff
@@ -0,0 +1,119 @@
+from django.db import models
+
+from users.models import User
+
+class Info(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'infos'
+
+class Category(models.Model):
+    name = models.CharField(max_length=45)
+    
+    class Meta:
+        db_table = 'categories'
+
+class RoomType(models.Model):
+    name = models.CharField(max_length=45)
+    
+    class Meta:
+        db_table = 'room_types'
+
+class Evaluate(models.Model):
+    name = models.CharField(max_length=45)
+    
+    class Meta:
+        db_table = 'evaluates'
```

Comment:
Class 명은 명사를 사용해서 만들어주세요~
(함수명은 동사를 사용합니다. 함수는 행동, 행위의 기능을 하기 때문이죠~)

```suggestion
class Evaluation(models.Model):
    name = models.CharField(max_length=45)
    
    class Meta:
        db_table = 'evaluates'
```

Reply:
넵 수정완료하였습니다